index.html

```diff
@@ -75,15 +143,33 @@ <h1>Mouse test</h1>
     });
 
     document.addEventListener("mousedown", event => {
-        switch (event.button) {
-            case 0: circle.classList.add("active-left"); break;
-            case 1: circle.classList.add("active-middle"); break;
-            case 2: circle.classList.add("active-right"); break;
-        }
+        circle.classList.add(`active-${event.button}`);
+
+        event.preventDefault();
+
+        const diff = performance.now() - lastClick[event.button];
```

Comment:
`lastClick[event.button]` is undefined the first time it runs. I think we want to protect the whole block here and do not log if there is no previous timestamp to compare against.